tests/processes/DimProcess.Library.Tests/TechnicalUserDeletionProcessHandlerTests.cs

```diff
@@ -0,0 +1,121 @@
+using Dim.Clients.Api.Cf;
```

Comment:
file header missing, also for migration files

Reply:
added